src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -70,6 +71,9 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
   def withRetry(work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit): Unit = {
     work onComplete {
       case Success(s) => onSuccess(s)
+      case Failure(e: GoogleJsonResponseException) if e.getStatusCode == 403 =>
+        logger.error(e.getMessage, e)
+        self ! Finish(FailedExecutionHandle(e))
```

Comment:
This looks good for now, we'll probably want to generalize this a bit more to handle "hard failures" as opposed to the "soft failures" below.  

I'd like to have some comments on both this and the `Error` case below so no one else is confused by what's happening here.  :smile: